src/client/scenes/mainScene.ts

```diff
@@ -67,45 +69,73 @@ export default class MainScene extends Phaser.Scene implements UpdateListener {
     return player
   }
 
-  update() {
-    let cursors = this.input.keyboard.createCursorKeys();
-    let sprite = this.players.get('self')
-
-    if (cursors.left.isDown) {
-      sprite.setVelocityX(-160);
-      sprite.anims.play('left', true);
-    } else if (cursors.right.isDown) {
-      sprite.setVelocityX(160);
-      sprite.anims.play('right', true);
-    } else {
-      sprite.setVelocityX(0);
-      sprite.anims.play('turn');
-    }
-
-    if (cursors.up.isDown && sprite.body.touching.down) {
-      sprite.setVelocityY(-330);
-    }
+  removePlayer(id: string) {
+    this.players.get(id).destroy()
+    this.players.delete(id)
+  }
 
-    if (sprite.x != this.prev_location[0] || sprite.y != this.prev_location[1]) {
-      this.socket.updatePos(sprite.x, sprite.y)
-      this.prev_location = [sprite.x, sprite.y]
+  update() {
+    let cursors = this.input.keyboard.createCursorKeys()
+    if (this.controlling_id) {
```

Comment:
Where is the closing curly bracket for this opening one?

Reply:
https://github.com/kyleai/socket-io-game/pull/7/files/861ea873ed278ce21644758d92272b60b4a1976e#r436321055

it checks whether we have been given a UUID from the server. if not, we don't want to move